module-4/backend/week_four.md

```diff
@@ -1,19 +1,29 @@
 ## Week Four - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this repository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's your favorite project management tool?
+  - Pivotal Tracker. I miss Waffle's integration with git issues, but  I think a lot more detail can go in to Pivotal cards.
```

Comment:
quite a bit of built in detail - but you could add more detail to waffle if you'd like (like tags, estimation counts, etc.)